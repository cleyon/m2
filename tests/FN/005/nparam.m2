@define fn I see $# args
0: @fn@
1: @fn foo@
2: @fn foo bar@
3: @fn foo bar baz@
4: @fn foo bar baz quux@
