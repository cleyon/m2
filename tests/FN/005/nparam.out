0: I see 0 args
1: I see 1 args
2: I see 2 args
3: I see 3 args
4: I see 4 args
